Codeinwp/visualizer
# Improve create chart workflow from Visualizer Block

### Summary


Ported the Chart Library workflow to the Visualizer Block for chart creation. Same workflow as in https://github.com/Codeinwp/visualizer/pull/1136

### Will affect visual aspect of the product

NO

### Screenshots 

https://github.com/Codeinwp/visualizer/assets/17597852/b44dd3ac-1fa2-4cd9-8dcb-5f690e04e1be


### Test instructions


- Insert a new visualizer block and click to create a new chart
- You should see the Popup
- After Creating the chart, the block will update with the newly created chart

## Check before Pull Request is ready:



Closes #1119
